tests/repos/datasets/test_dataset_concepticon_concepts/tdcc.py

```diff
@@ -0,0 +1,15 @@
+import pathlib
+
+from pylexibank import Dataset, CONCEPTICON_CONCEPTS
+
+
+class Test(Dataset):
+    dir = pathlib.Path(__file__).parent
+    id = 'test_dataset_concepticon_concepts'
+    concept_class = CONCEPTICON_CONCEPTS
```

Comment:
So this is how you mark a dataset as one where the concept class is created from the concepticon list.